cookie_monster/static/cookie_monster/css/styles.css

```
@@ -1,42 +1,47 @@
-.cookieMakerContainer .cookieMakerWarning {
-  color: #b94a48;
-  background-color: #f2dede;
-  border-color: #eed3d7;
+.cookie-monster-banner {
+    display: none;
+    width: auto;
+    min-height: 20px;
+    padding: 30px 30px 30px 30px;
```

This would be a bit cleaner
```suggestion
    padding: 30px;
```